recipes/opencv/4.x/conanfile.py

```diff
@@ -49,6 +50,7 @@ class OpenCVConan(ConanFile):
         "contrib": False,
         "contrib_freetype": False,
         "contrib_sfm": False,
+        "with_ade": True,
```

Comment:
@madebr what is the rationale when choosing to make an option enabled by default? I am making this as true so that the tests are run with this option. Do the tests run with all the variations of the options?

Reply:
Usually, we model the options such that a build without option overrides has the same option values as the project's build script.